### PR TITLE
Add emergency application flow to integration tests

### PR DESCRIPTION
**Pull request by richpjames**
This isn't exhaustive as I wanted to avoid duplicated what we already have in the long, slow full Apply test but it covers the new screens.

### CODE REVIEW

File: server/testutils/addResponseToApplication.ts [hunk 1]
```diff
@@ -2,15 +2,31 @@ import { ApprovedPremisesApplication as Application } from '../@types/shared'
 
 export const addResponseToApplication = (
```
Comment (pezholio) [inline server/testutils/addResponseToApplication.ts:3]: I wonder if this should be broken into two and have an `addResponsesToApplication` that accepts an object, to save the duplication and conditionals.

File: integration_tests/tests/apply/apply.cy.ts [hunk 2]
```diff
@@ -122,7 +121,7 @@ context('Apply', () => {
     isExceptionalCasePage.clickSubmit()
 
     // Then I should be told the application is not eligible
-    Page.verifyOnPage(NotEligiblePage)
+    // Page.verifyOnPage(NotEligiblePage)
```
Comment (pezholio) [inline integration_tests/tests/apply/apply.cy.ts:124]: Why is this commented out?

Comment (richpjames) [reply integration_tests/tests/apply/apply.cy.ts]: No good reason, fixed now

File: server/testutils/addResponseToApplication.ts (hunk 1 above)
Comment (richpjames) [reply server/testutils/addResponseToApplication.ts]: Yeah, seems reasonable

**Review by pezholio — APPROVED** [review]
LGTM 👍

**Pull request merged**